nodejs/eks/nodegroup.ts

```diff
@@ -1836,9 +1839,20 @@ Content-Type: text/x-shellscript; charset="us-ascii"
         ? { httpTokens: "required", httpPutResponseHopLimit: 2, httpEndpoint: "enabled" }
         : undefined;
 
+    const blockDeviceMappings = [
+        {
+            deviceName: "/dev/xvda",
+            ebs: {
+                volumeSize: args.diskSize,
+                volumeType: "gp2"
```

Comment:
You could remove the `volumeType` here. That way it'll use the AMIs default volume type